WordPress/UITestsFoundation/Screens/Me/SupportScreen.swift

```diff
@@ -9,52 +9,57 @@ public class SupportScreen: ScreenObject {
         $0.buttons["close-button"]
     }
 
-    private let contactSupportButtonGetter: (XCUIApplication) -> XCUIElement = {
-        $0.cells["contact-support-button"]
-    }
-
-    private let contactEmailFieldGetter: (XCUIApplication) -> XCUIElement = {
-        $0.textFields["Email"]
+    private let visitForumsButtonGetter: (XCUIApplication) -> XCUIElement = {
+        $0.cells["visit-wordpress-forums-button"]
     }
 
     private let okButtonGetter: (XCUIApplication) -> XCUIElement = {
         $0.buttons["OK"]
     }
 
     var closeButton: XCUIElement { closeButtonGetter(app) }
-    var contactSupportButton: XCUIElement { contactSupportButtonGetter(app) }
-    var contactEmailField: XCUIElement { contactEmailFieldGetter(app) }
+    var visitForumsButton: XCUIElement { visitForumsButtonGetter(app) }
     var okButton: XCUIElement { okButtonGetter(app) }
 
     public init(app: XCUIApplication = XCUIApplication()) throws {
         try super.init(
             expectedElementGetters: [
                 closeButtonGetter,
-                contactSupportButtonGetter,
+                visitForumsButtonGetter,
                 // swiftlint:disable opening_brace
-                { $0.cells["help-center-link-button"] },
-                { $0.cells["my-tickets-button"] },
-                { $0.cells["set-contact-email-button"] },
+                { $0.cells["visit-wordpress-forums-prompt"] },
                 { $0.cells["activity-logs-button"] }
                 // swiftlint:enable opening_brace
             ],
             app: app,
-            waitTimeout: 7
+            waitTimeout: 22
         )
     }
 
-    public func contactSupport() throws -> ContactUsScreen {
-        contactSupportButton.tap()
-        addContactInformationIfNeeded()
-        return try ContactUsScreen()
+    public func assertVisitForumButtonEnabled() -> SupportScreen {
+        XCTAssert(visitForumsButton.isEnabled)
+        return self
     }
 
-    private func addContactInformationIfNeeded() {
-        if contactEmailField.waitForExistence(timeout: 3) {
-            contactEmailField.tap()
-            contactEmailField.typeText("user@test.zzz")
-            okButton.tap()
+    public func visitForums() {
+        let safari = XCUIApplication(bundleIdentifier: "com.apple.mobilesafari")
+        visitForumsButton.tap()
+
+        // Select the Address bar when Safari opens
+        let addressBar = safari.textFields["Address"]
+        guard addressBar.waitForExistence(timeout: 5) else {
+            XCTFail("Address bar not found")
+            return
         }
+        addressBar.tap()
+
+        guard safari.wait(for: .runningForeground, timeout: 4) else {
+            XCTFail("Safari wait failed")
+            return
+        }
+
+        XCTAssertEqual(addressBar.value as! String, "https://wordpress.org/support/forum/mobile/")
+        app.activate() //Back to app
```

Comment:
WDYT of extracting this part into a separate function called `assertForumsLoaded` (just a name example). This way, by looking into the test code, it will be clear what is going on behind the scenes (more obvious "action followed by an assertion" structure)

```swift
        try PrologueScreen()
            .selectContinue()
            .selectHelp()
            .assertVisitForumButtonEnabled()
            .visitForums()
            .assertForumsLoaded()
```

Reply:
Sounds great and will do! 👍